### ISSUE TITLE
DropDowns ComboBox : Keyboard Navigation 

### ISSUE THREAD

**Issue by ZiedHf** [telerik/kendo-react#90]
According to the documentation here : [link](https://www.telerik.com/kendo-react-ui/components/dropdowns/combobox/keyboard-navigation/)

I'm expecting from the example of the link above to : 
1- Open the _Popup_
2- Select an item, "_Baseball_" for example.
3- Open the _Popup_ again
4- Navigate using **Up and Down Arrows**
5- Hit **ESC**
6- The item "_Baseball_" should be selected, unless I hit **Enter** in step 5.

The documentation stresses that the Up arrow : "_Selects the previous available item._" which is right. The second part is incorrect "_Highlights it only if the popup is open._" I believe it's a bug here and should be fixed.

**Comment by nstoychev** [comment]
By design, the ComboBox behaves similar to [native select](https://codepen.io/gaearon/pen/JbbEzX?editors=0010) element which includes the keyboard navigation. 

"Highlights it only if the popup is open." will be removed.

**Comment by ZiedHf** [comment]
In the example of [native select](https://codepen.io/gaearon/pen/JbbEzX?editors=0010) the _arrow keys_ up and down will **NOT** trigger the _onChange_ function unless you hit _enter_ to change the current value. _ESC_ will close the Popup without changing anything. This is awesome.

On the contrary of the **ComboBox** component, the same action will change the value whenever you hit the _arrow keys_. _Enter_ and _ESC_ are useless except for closing the Popup which is weird.

I'm using the **ComboBox** in my App and I can't navigate using the _arrow keys_ because the _onChange_ function contains several Back-End calls. Navigation without changing here is impossible and it will be a time consuming.

I think making the **ComboBox** behaves like the example of [native select](https://codepen.io/gaearon/pen/JbbEzX?editors=0010) is necessary.

**Comment by nstoychev** [comment]
Thank you for the feedback.

I have checked the [native select](https://codepen.io/gaearon/pen/JbbEzX?editors=0010) example and found out that _ESC_ key also triggers onChange event.

In your App, can you use the ComboBox [onChange event](https://www.telerik.com/kendo-react-ui/components/dropdowns/api/ComboBoxChangeEvent/) object which contains the native and synthetic events, and skip the Back-End calls when event is triggered by arrow keys?

Here is an example: https://stackblitz.com/edit/react-8ptytd?file=app/main.js

**Comment by kdikov82** [comment]
Won`t fix: Will introduce major breaking change for existing projects.
